vhs-frontend/.eslintrc.json

```diff
@@ -44,6 +44,6 @@
       }
     ],
     "no-undef": "off",
-    "@typescript-eslint/no-unused-vars": ["error"]
+    "@typescript-eslint/no-unused-vars": ["warn"]
```

Comment:
This is completely fine. Depending on how strict we want to be, I sometimes do put error here. Warn might be better if you're doing a lot of experimentation and leave some variables for later and commit them unused because you're changing stuff all the time.

At the end of the project it's good to put a stricter set of rules in eslint so that you can evaluate the project for production better.

You don't have to change this, I'm just commenting. :)